online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZSpinner.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 23/8/2022.
+ */
+
+package com.adyen.checkout.onlinebankingcz
+
+import android.content.Context
+import android.net.Uri
+import android.util.AttributeSet
+import android.view.View
+import androidx.browser.customtabs.CustomTabsIntent
+import com.adyen.checkout.components.model.payments.request.OnlineBankingCZPaymentMethod
+import com.adyen.checkout.components.ui.util.ThemeUtil
+import com.adyen.checkout.issuerlist.IssuerListSpinnerView
+
+class OnlineBankingCZSpinner @JvmOverloads constructor(
```

Comment:
All views are suffixed with `View`, so it should be `OnlineBankingCZSpinnerView`